rmgpy/rmg/input.py

```diff
@@ -1526,7 +1528,8 @@ def read_input_file(path, rmg0):
         'SMILES': smiles,
         'InChI': inchi,
         'adjacencyList': adjacency_list,
-        'adjacencyListGroup': adjacency_list_group,
+        'adjacencyListGroup': group_adjacency_list,
```

Comment:
We should add to one of the example input files, and documentation.

Reply:
Instead I removed this feature, because it already exists in https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2185